server/storages/bigquery.go

```diff
@@ -71,7 +71,7 @@ func NewBigQuery(config *Config) (Storage, error) {
 		return nil, err
 	}
 
-	tableHelper := NewTableHelper(bigQueryAdapter, config.monitorKeeper, config.pkFields, adapters.SchemaToBigQueryString, config.streamMode)
+	tableHelper := NewTableHelper(bigQueryAdapter, config.monitorKeeper, config.pkFields, adapters.SchemaToBigQueryString, config.streamMode, config.max_columns)
 
```

Comment:
Shouldn't it be maxColumns instead of max_columns?

Reply:
Absolutely

Reply:
Renamed